manim/mobject/graphing/functions.py

```diff
@@ -158,7 +158,7 @@ def generate_points(self):
                 x, y, z = self.function(t_range)
                 if not isinstance(z, np.ndarray):
                     z = np.zeros_like(x)
-                points = np.stack(zip(x, y, z), axis=0)
+                points = np.stack([x, y, z], axis=1)
```

Comment:
Any reason why you changed the `axis` parameter here?

Reply:
The list does not return the same thing as the zip, one have to flip the axis for the same output.

But! Don't take my word for it, in the related Issue there is a code snippet you can put in a Scene, debug and test. If you are using VS Code I have a launch.json debug config you can use.

Reply:
Here
```json
{
    "version": "0.2.0",
    "configurations": [
        {
            "name": "Python: Current File",
            "type": "python",
            "request": "launch",
            "program": "manim.py ${file}",
            "console": "integratedTerminal"
        }
    ]
}
```

Reply:
Tested with a small example,
```pycon
>>> x = np.linspace(0, 5, 10)
>>> y = x**2
>>> z = x*0.1 + 1
>>> np.all(np.stack(zip(x, y, z), axis=0) == np.stack([x, y, z], axis=1))
True
```

Reply:
Oh right, I didn't notice that. Thanks!